Scripts/script-CBSearch.yml

```diff
@@ -49,7 +49,10 @@ script: |-
       };
       if(args.query)
           myArgs.query = args.query;
-      res = executeCommand('cb-' + searchType, myArgs);
+          if(searchType == 'binary')
+              res = executeCommand('cb-' + searchType, myArgs);
```

Comment:
if we already know the the command name, no need for the string concat here, just execute the command name